cli/command/engine/activate_windows.go

```diff
@@ -0,0 +1,9 @@
+// +build windows
+
+package engine
+
+var (
+	isRoot = func() bool {
```

Comment:
We'd like to support this on windows in a future release.

Reply:
Yeah, but it's not there. Coding "because someday it may be needed" is resulting can lead to many issues. We don't know _when_ it arrives, and if the implementation will be as we anticipate it to be right now. Not a blocker for me, but definitely not a fan.